index.js

```diff
@@ -1 +1,17 @@
-﻿
\ No newline at end of file
+import './modules/button_functions.js';
+import DateTimeFormat from './modules/dateTime.js';
+import { addBooks } from './modules/add_remove.js';
+import { CheckLocalStorage, booksArr } from './modules/onload.js';
+import './modules/extrafunctions.js';
+
+const displayTime = () => setInterval(() => {
+  document.querySelector('#date').innerHTML = DateTimeFormat();
+}, 1000);
```

Comment:
- [ ] The position of the date on the page is not properly positioned. Kindly fix that.

[app-date.webm](https://user-images.githubusercontent.com/26597432/206987565-aef5f3fd-f9bd-467d-842a-b279d8510faf.webm)

Reply:
Position fixed. Thank you